src/pipeline.cc

```diff
@@ -1330,6 +1336,9 @@ Napi::Value pipeline(const Napi::CallbackInfo& info) {
   baton->linearB = sharp::AttrAsDouble(options, "linearB");
   baton->greyscale = sharp::AttrAsBool(options, "greyscale");
   baton->normalise = sharp::AttrAsBool(options, "normalise");
+  baton->claheWidth = sharp::AttrAsInt32(options, "claheWidth");
```

Comment:
Can use the unsigned `AttrAsUint32` for dimensions as they will always be positive.